apps/portal/components/pagesComponents/integrations/Slack.tsx

```
@@ -47,39 +49,37 @@ const Slack: React.FC<ISettingsPageProps> = () => {
             path="/settings/slack-icon.svg"
             size="22px"
           />
-          Slack Integration
+          {t('page.integrations.slack.title')}
         </div>
       </h1>
       {user && user.isSlackIntegrate && user.isSlackIntegrate === true ? (
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)